hubblestack_nova_profiles/top.nova

```diff
@@ -11,6 +11,7 @@ nova:
     - cis.centos-6-level-1-scored-v2-0-1
   'G@osfinger:CentOS*Linux-7':
     - cis.centos-7-level-1-scored-v2-1-0
+    - control.centos-7
```

Comment:
Can you remove this line? I think it might be leftover from you writing your own control files, but there's no `control` directory that ships with the default hubble profiles.